### PR TITLE
Improve default UI

### PR DESCRIPTION
**Pull request by axe312ger**

### CODE REVIEW

File: packages/core/src/integration-helpers.tsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ export const createIconComponentFromSimpleIconsSvgPath: (
     xmlns="http://www.w3.org/2000/svg"
     fill={color}
     viewBox="0 0 24 24"
+    height="100%"
```
Comment (axe312ger) [inline packages/core/src/integration-helpers.tsx:21]: @todo add comment that this is required for FF & Safari to properly render SVGs. Alternative: via CSS

File: packages/docs/src/theme/Root.js [hunk 2]
```diff
@@ -16,9 +16,10 @@ const Button = props => (
   <button
     type="submit"
     className="button button--primary button--block"
+    style={{ marginBottom: '1rem' }}
     {...props}
   >
-    Save
+    {props.children}
```
Comment (axe312ger) [inline packages/docs/src/theme/Root.js:22]: @todo not needed as we spread props above. Refactor to  self closing component

File: packages/interface-default/src/backdrop.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
+// @todo hide toggle button via animation, use only big buttons to save & close
```
Comment (axe312ger) [inline packages/interface-default/src/backdrop.tsx:1]: @todo remove copy pasta comments

File: packages/interface-default/src/index.tsx [hunk 4]
```diff
@@ -122,47 +155,59 @@ export const UnobtrusiveConsentControlUI: React.FC<UnobtrusiveConsentControlUIPr
...
-                    <SubmitButton>Save</SubmitButton>
-                  </div>
+      {renderBackdrop && (
+        <Backdrop
+          show={showForm}
+          fadeDuration={slideDuration}
+          styles={styles}
+        />
+      )}
+      <CSSTransition
```
Comment (axe312ger) [inline packages/interface-default/src/index.tsx:165]: I guess its time to make the whole form a component and make it overridable like the others

File: packages/interface-default/src/animation-blur.module.css [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+.enterActive {
+  backdrop-filter: blur(4px);
+  transition-property: backdrop-filter;
+  transition-timing-function: ease-in-out;
+}
+
+.enterDone {
+  backdrop-filter: blur(4px);
```
Comment (axe312ger) [inline packages/interface-default/src/animation-blur.module.css:12]: @todo revert the animation logic:

css of the animated element should have visible status by default, the animation then adds the in/out transitions.

That way we do not need the enterDone class

**Review by zcei — COMMENTED** [review]
Merge on your own behalf 🤷‍♂️

File: packages/interface-default/src/animation-blur.module.css (hunk 5 above)
Comment (axe312ger) [reply packages/interface-default/src/animation-blur.module.css:12]: na that might make it more complicated or result in flickering elements. Lets keep it

**Pull request merged**